contracts/utils/TablelandDeployments.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "../ITablelandTables.sol";
+
+/**
+ * @dev Helper library for getting an instance of ITablelandTables for the currently executing EVM chain.
+ */
+library TablelandDeployments {
+    /**
+     * Current chain does not have a TablelandTables deployment.
+     */
+    error ChainNotSupported(uint256 chainid);
+
+    // TablelandTables address on Ethereum Goerli.
+    address internal constant GOERLI =
+        0xDA8EA22d092307874f30A1F277D1388dca0BA97a;
+    // TablelandTables address on Optimism Kovan.
+    address internal constant OPTIMISTIC_KOVAN =
+        0xf2C9Fc73884A9c6e6Db58778176Ab67989139D06;
+    // TablelandTables address on Polygon Mumbai.
+    address internal constant POLYGON_MUMBAI =
+        0x4b48841d4b32C4650E4ABc117A03FE8B51f38F68;
+
+    /**
+     * @dev Returns an interface to Tableland for the currently executing EVM chain.
+     *
+     * Requirements:
+     *
+     * - block.chainid must refer to a supported chain.
+     */
+    function get() internal view returns (ITablelandTables) {
```

Comment:
@andrewxhill see here... turns out you can get the chain id from the block, so calling `get()` will just return you the correct contract.

Reply:
oh neat. TIL